.devcontainer/DOCKERFILE

```diff
@@ -0,0 +1,6 @@
+FROM mcr.microsoft.com/vscode/devcontainers/rust:1-bullseye
+
+# Install clang
+RUN apt-get update \
+  && apt-get install -y clang \
```

Comment:
do we need clang?

Reply:
Yeah I think it's a good idea. Bindgen and other things need it, I think.